frontend/src/metabase/query_builder/components/filters/modals/InlineCategoryPicker/SimpleCategoryFilterPicker.tsx

```diff
@@ -31,20 +36,21 @@ export function SimpleCategoryFilterPicker({
   };
 
   const hasShortOptions = !options.find(
-    option => String(option).length > LONG_OPTION_LENGTH,
+    ([value, displayValue]) =>
```

Comment:
I know the PR is merged but I noticed the variable name doesn't match the one on line 48. `value` vs `option` when both are destructured from the same `options`.